testing/events.go

```diff
@@ -12,7 +13,7 @@ import (
 	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
 )
 
-type EventsMap map[string]map[string]string
+type EventsMap map[string][]map[string]string
```

Comment:
I wonder if we need this type, its kind of complex. I would probably opt for just using the `sdk.Events` type. 
Can't we just write the events assertion like this:

```go
events := res.GetEvents()
expected := sdk.Events{
	sdk.NewEvent(
		types.EventTypeDistributeFee,
		sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainA.SenderAccount.GetAddress().String()),
		sdk.NewAttribute(types.AttributeKeyFee, defaultRecvFee.String()),
	),
	sdk.NewEvent(
		types.EventTypeDistributeFee,
		sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainA.SenderAccount.GetAddress().String()),
		sdk.NewAttribute(types.AttributeKeyFee, defaultAckFee.String()),
	),
	sdk.NewEvent(
		types.EventTypeDistributeFee,
		sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainA.SenderAccount.GetAddress().String()),
		sdk.NewAttribute(types.AttributeKeyFee, defaultTimeoutFee.String()),
	),
}

for _, evt := range expected {
	suite.Require().Contains(events, evt)
}
```

Reply:
Great idea! Much simpler! :)